x11-misc/qt5ct/files/qt5ct-1.9-no-qtpaths.patch

```diff
@@ -0,0 +1,29 @@
+--- a/CMakeLists.txt	2025-01-27 19:16:40.000000000 +0100
```

Comment:
Could you send that upstream or at least give me a commit message/explanation I could pass on to upstream?

Reply:
(made it a more upstreamable patch, not going to do it myself though)